hbase-server/src/main/java/org/apache/hadoop/hbase/replication/HBaseReplicationEndpoint.java

```diff
@@ -191,6 +196,17 @@ private synchronized void reloadZkWatcher() throws IOException {
     zkw.registerListener(new PeerRegionServerListener(this));
   }
 
+  private synchronized void connectPeerCluster() throws IOException {
```

Comment:
Why synchronized here?

Reply:
To avoid SpotBugs Inconsistent synchronization about accessing `conn`